src/test/groovy/com/github/maleksandrowicz93/cqrsdemo/student/StudentUtils.groovy

```diff
@@ -0,0 +1,84 @@
+package com.github.maleksandrowicz93.cqrsdemo.student
+
+import com.github.maleksandrowicz93.cqrsdemo.student.dto.AddStudentCommand
+import com.github.maleksandrowicz93.cqrsdemo.student.dto.EditStudentDataCommand
+import com.github.maleksandrowicz93.cqrsdemo.student.dto.StudentDto
+
+import java.time.LocalDate
+import java.time.Month
+
+class StudentUtils {
+
+    static final EMAIL = "john.paul@gmail.com"
+    static final FIRST_NAME = "John"
+    static final LAST_NAME = "Paul"
+    static final PASSWORD = "password123"
+    static final BIRTH_DATE = LocalDate.of(1993, Month.AUGUST, 21)
+
+    static final ALTERNATIVE_EMAIL = "matty.cash@gmail.com"
```

Comment:
jakby doszedł trzeci, to byłby `YET_ANOTHER_EMAIL`? :p propozycja:
```java
class Students {

    static class First {
        
        static final EMAIL = "...";
    }

    static class Second {
        
        static final EMAIL = "...";
    }

}

Reply:
mega patent! widzę też, że ładnie można opędzić to enumem xD

Reply:
Jak najbardziej można i wtedy powodzenia z inicjalizacją w odpowiedniej kolejności, jakby zależały od siebie XD Wyleczymy Cię jeszcze z tych enumów 😎